node_package/src/buildConsoleReplay.js

```diff
@@ -5,6 +5,7 @@ import scriptSanitizedVal from './scriptSanitizedVal';
 
 export function consoleReplay() {
   // console.history is a global polyfill used in server rendering.
+  // $FlowFixMe
```

Comment:
As per [this flow article](https://medium.com/flow-type/new-flow-errors-on-unknown-property-access-in-conditionals-461da66ea10), flow is now throwing errors for unknown property access in conditionals. I flagged this with `$FlowFixMe` for the time being to suppress the error. @justin808 what are your thoughts on fixing this?